x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/index.tsx

```diff
@@ -406,6 +406,15 @@ export const CreatePackagePolicySinglePage: CreatePackagePolicyParams = ({
                 handleSetupTechnologyChange(value);
                 // agentless doesn't need system integration
                 setWithSysMonitoring(value === SetupTechnology.AGENT_BASED);
+
+                // reset selected output if switching to agentless
```

Comment:
It is not confusing for the user? especially as the order of these inputs in the UI is selecting output first before agentless right?

Reply:
I removed this and moved the resetting logic elsewhere that only triggers if the selected output is not compatible